.travis.yml

```diff
@@ -39,6 +39,11 @@ script:
   - sudo make -C mtf/metadata check
   - sudo make -C examples/testing-module check-mtf-metadata
   - sudo make -C examples/testing-module check-openshift
-  #- sudo make -C examples/testing-module check-mtf-pdc-module-info-reader
+  - sudo make -C examples/testing-module check-mtf-pdc-module-info-reader
+  - sudo make -C examples/s2i-python-sample check
+  - sudo make -C examples/s2i-python-sample check-python-3.5
+  - sudo mtf -l && false || true
+  - sudo mtf -l --url fedora && false || true
+  - sudo mtf -l --url fedora --module docker
 
 after_script: sudo cat /home/travis/avocado/job-results/latest/job.log
```

Comment:
It would be nice to collect all logs from all tests. Nowdays, only the latest is visible in 'raw log'

Reply:
There is problem with travis, that this raw log is limited, and unable to show all logs there and also unable to store logs in travis, or at least I don't know how to do

Reply:
It is impossible. or at least I don't know how  to do it in travis, would be nice to store all logs.
RAW travis log has limited size, display all logs there will invoke travis troubles (I'me met this issue few times when we've added --show-job-log there by default)